### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by ice-orion**

### CODE REVIEW

**Review by ice-ares — APPROVED** [review]

File: src/navigation/components/MainTabBar/components/MiningTooltip/components/PreStakingInfo.tsx [hunk 1]
```diff
@@ -77,7 +79,11 @@ export const PreStakingInfo = () => {
           height={rem(18)}
         />
         <Text style={styles.bonusLabelText}>{t('staking.bonus_label')}</Text>
-        <Text style={styles.bonusValueText}>+{preStakingSummary?.bonus}%</Text>
+        {miningRates && (
+          <Text style={styles.bonusValueText}>
+            +{miningRates.total.bonuses?.extra}%
```
Comment (ice-hades) [inline src/navigation/components/MainTabBar/components/MiningTooltip/components/PreStakingInfo.tsx:84]: Should you return 0 here as a default?

**Review by ice-hector — DISMISSED** [review]

File: src/screens/HomeFlow/Home/components/Pager/components/MiningRate/index.tsx [hunk 2]
```diff
@@ -26,9 +26,6 @@ export const MiningRate = memo(() => {
     return null;
   }
 
-  const miningSign =
```
Comment (ice-hades) [inline src/screens/HomeFlow/Home/components/Pager/components/MiningRate/index.tsx:29]: why inlined this constant?

Comment (ice-orion) [reply src/screens/HomeFlow/Home/components/Pager/components/MiningRate/index.tsx:29]: now the values are a bit different in those places where it was used

File: src/navigation/components/MainTabBar/components/MiningTooltip/components/PreStakingInfo.tsx (hunk 1 above)
Comment (ice-orion) [reply src/navigation/components/MainTabBar/components/MiningTooltip/components/PreStakingInfo.tsx]: yea, added 0 and i picked the wrong values there -> `preStaking` should have been used

**Review by ice-hades — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Review by ice-ares — APPROVED** [review]

**Pull request merged**